ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-main.yaml

```diff
@@ -63,7 +49,7 @@ tests:
     make test
   container:
     from: src
-- as: e2e-aws-operator
+- as: e2e-aws-ovn-operator
```

Comment:
The *-operator jobs do not need to have ovn added.

Reply:
Done.